microsoftgraph/microsoft-graph-explorer-v4
# Task: Add browser storage for resources

## Overview

Closes #2369 
### Demo
- When the first fetch is unsuccessful, and the storage instance is empty, this is displayed to the user:
![image](https://user-images.githubusercontent.com/34104277/215984987-3256b753-91da-4b86-974e-c1b177204d33.png)

- Otherwise, the resources are displayed (served from DevX API or the browser storage when DevX API is down).

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout to this branch.
* Run the local instance of GE.
* Navigate to the Resources tab.
* Observe changes.
